app/views/recipes/_form.html.erb

```diff
@@ -13,33 +10,47 @@
         </div>
       </div>
 
-      <div class="relative">
-        <div class="absolute inset-0 flex items-center" aria-hidden="true">
-          <div class="w-full border-t border-gray-300"></div>
-        </div>
-        <div class="relative flex justify-start">
-          <span class="bg-white pr-2 text-sm text-gray-500">Ingredients</span>
-        </div>
-      </div>
+      <div data-controller="template-manager">
```

Comment:
The template manager controller is only in charge of knowing where the template is and where it should go in the DOM once an action is taken.

In this example, the action is clicking the + button